src/index.js

```diff
@@ -61,7 +61,7 @@ const App = () => {
         setToDos(toRemainItems);
     }
 
-    const toggleSort = (columnToSort) => {
+    const toggleSortColumn = (columnToSort) => {
```

Comment:
Same here with the naming. This name isn't as bad as previous one but it makes me feel like you're toggling a "sort item". Having two verb grammarly doesn't make sense, at least it's not straightforward. 

You can just name it `onSortColumn` or `onSort` since this is what the function is actually doing. "toggle" doesn't really mean anything here because that's a bit too much information for the user to read. 
No need to go with what I suggested. Just don't use the two verb system and I think you'll be fine. 😉